lib/serverless-tools/deployer/python_builder.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module ServerlessTools
+  module Deployer
+    class PythonBuilder
+      def initialize(config:)
+        @config = config
+      end
+      # Run three commands to build the python bundle for Lambda
+      def build
+        # Poetry does not have an option to install dependencies to a specified target folder.
+        # Workaround is generating a requirments.txt file using poetry
+        `poetry export -f requirements.txt --without-hashes > requirements.txt`
+        # And then installing them using pip to specified "bundle" target directory
+        `pip install -r requirements.txt -t ./bundle/`
```

Comment:
I wonder if we should use 'package' instead of 'bundle' here? Just to avoid any confusion if people think this is somehow related to bundler.
Also, the docs use package: https://docs.aws.amazon.com/lambda/latest/dg/python-package.html

Reply:
Yes that will definitely help avoid confusion!